app/controllers/all/export/grant_management_and_finance_unit/transfers/projects_controller.rb

```diff
@@ -0,0 +1,42 @@
+class All::Export::GrantManagementAndFinanceUnit::Transfers::ProjectsController < ApplicationController
+  def index
+    authorize Project, :index?
+
+    months = []
+    export_months.each do |month|
+      months << {
+        month: month,
+        count: ProjectsForExportService.new.grant_management_and_finance_unit_transfer_projects(month: month.month, year: month.year).count
+      }
+    end
+    @data = months
+  end
+
+  def show
+    authorize Project, :show?
+    @month = Date.parse("#{year}-#{month}-1")
+  end
+
+  def csv
+    authorize Project, :index?
+
+    projects = ProjectsForExportService.new.grant_management_and_finance_unit_transfer_projects(month: month, year: year)
+    csv = Export::Transfers::GrantManagementAndFinanceUnitCsvExportService.new(projects).call
+
+    send_data csv, filename: "#{year}-#{month}_grant_management_and_finance_unit_transfers_export.csv", type: :csv, disposition: "attachment"
+  end
+
+  private def month
+    params[:month]
```

Comment:
Maybe do the `.to_i` in my suggestion here instead? Depending on if you want a string or an int, generally.